rotkehlchen/tasks/manager.py

```diff
@@ -77,8 +80,10 @@ def __init__(
             exchange_manager: ExchangeManager,
             evm_tx_decoder: 'EVMTransactionDecoder',
             eth_transactions: 'EthTransactions',
-            deactivate_premium: Callable,
+            deactivate_premium: Callable[[], None],
+            activate_premium: Callable[[Premium], None],
             query_balances: Callable,
+            rotki_notifier: 'RotkiNotifier' = None,
```

Comment:
Why is a default argument given here? When can it be None?

Reply:
I was trying to go fast and set it to `None` just to be sure. Is only initialized in the rotkehlchen class (where I pass the instance) and in the tests

Reply:
Setting something to None is not going fast, it is asking for trouble.